src/default-settings.js

```diff
@@ -1,167 +1,177 @@
+                ,'stroke-width': 2}}
+        ]
+    ,legend:
+        {location: 'top'
+        ,label: 'Severity'}
+    ,color_by: null // set in syncSettings()
+    ,colors:
+        ['#66bd63'
+        ,'#fdae61'
+        ,'#d73027']
+    ,date_format: '%Y-%m-%d'
+    ,y_behavior: 'flex'
+    ,gridlines: 'y'
+    ,no_text_size: false
+    ,range_band: 15
+    ,margin: {top: 50}
+    ,resizable: true
 };
 
-export function syncSettings(preSettings){
+export function syncSettings(preSettings) {
     const nextSettings = Object.create(preSettings);
+
+    if (!nextSettings.filter_cols || nextSettings.filter_cols.length === 0)
+        nextSettings.filter_cols =
+            [nextSettings.ser_col
+            ,nextSettings.sev_col
+            ,nextSettings.id_col
+            ,'Participant Status'];
+
     nextSettings.y.column = nextSettings.id_col;
+
     nextSettings.marks[0].per = [nextSettings.id_col, nextSettings.seq_col];
-    nextSettings.marks[0].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[0].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+
     nextSettings.marks[1].per = [nextSettings.id_col, nextSettings.seq_col, 'wc_value'];
-    nextSettings.marks[1].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[1].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+    nextSettings.marks[1].values = {wc_category: [nextSettings.stdy_col]};
+
     nextSettings.marks[2].per = [nextSettings.id_col, nextSettings.seq_col];
-    nextSettings.marks[2].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[2].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+    nextSettings.marks[2].values[nextSettings.ser_col] = ['Yes', 'Y'];
+
     nextSettings.marks[3].per = [nextSettings.id_col, nextSettings.seq_col, 'wc_value'];
-    nextSettings.marks[3].tooltip = `System Organ Class: [${nextSettings.soc_col}]\nPreferred Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`;
+    nextSettings.marks[3].tooltip = `Verbatim Term: [${nextSettings.term_col}]\nStart Day: [${nextSettings.stdy_col}]\nStop Day: [${nextSettings.endy_col}]`
+    nextSettings.marks[3].values = {wc_category: [nextSettings.stdy_col]};
+    nextSettings.marks[3].values[nextSettings.ser_col] = ['Yes', 'Y'];
+
```

Comment:
so `sev_vals` sets the order of axis. default is ['MILD','MODERATE','SEVERE'], which is reasonable, but we probably won't get close to 100% compliance there. What happens if there are other values or the formatting is different ("Mild" instead of "MILD")? Guessing they'll just get tacked on to the legend in alpha order?